### PR TITLE
Expression for trustyai

### PR DESCRIPTION
**Pull request by RHRolun**

### CODE REVIEW

File: charts/monitoring/templates/prometheusrules.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ spec:
     - alert: {{ .Values.name }}-datadrift-alert
       annotations:
         message: '{{ .Values.name }} has data drift...'
```
Comment (ckavili) [inline charts/monitoring/templates/prometheusrules.yaml:13]:
```suggestion
        message: '{{ .Values.name }} - Data drift detected.'
```

Comment (ckavili) [reply charts/monitoring/templates/prometheusrules.yaml:13]: actually, what would be a better message here @RHRolun 😅

**Review by ckavili — COMMENTED** [review]
better comment for the alert needed I think 🙈

File: charts/monitoring/templates/prometheusrules.yaml (hunk 1 above)
Comment (RHRolun) [reply charts/monitoring/templates/prometheusrules.yaml]: Message updated, lmk what you think :)

Comment (ckavili) [reply charts/monitoring/templates/prometheusrules.yaml]: awesome! thanks Robert!

**Pull request merged**